### DISCUSSION TITLE
[FR] respect the default folder setting for new notes

### DISCUSSION THREAD

**Discussion by huyz** [coddingtonbear/obsidian-web#59] [Q&A] [answered]
In core, I have "Folder to create new notes in" set. Can this plugin respect it? Or at least give an option in the extension.

**Comment by coddingtonbear** [comment, marked as answer]
You have control over this already.  Just adjust the "Path" field on whichever template you're using.  See the docs for https://github.com/coddingtonbear/obsidian-local-rest-api more information.
